src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,26 +40,135 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = if b then false else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean =

```

Comment:
I believe you can optimise (simplify) that. Hint -- if `left` is true what argument is responsible for the result?